builkio: Annotate backups with scheduling information.

miretskiy opened the pull request:
Annotate backup statement with scheduling information
when executing backups.

Release Notes: None

Commit:
builkio: Annotate backups with scheduling information.

Annotate backup statement with scheduling information
when executing backups.

Release Notes: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/51589)
<!-- Reviewable:end -->

miretskiy commented:
bors r+

Commit:
builkio: Improve error messages

Fix error message and add an additional check to ensure
that at least some schedule fields are initialized when
calling InitFromDatumns.

Release Notes: None

miretskiy commented:
bors r+
